lib/install/pllib/pl_lib.go

```diff
@@ -0,0 +1,316 @@
+// Copyright 2016-2017 VMware, Inc. All Rights Reserved.
```

Comment:
initially I named it as plclient, but later, I figured out portlayer function is not enough for vic-machine, so it's changed to library, cause a few server functions added